tests/phpunit/tests/general/template.php

```diff
@@ -421,6 +421,8 @@ private function set_custom_logo() {
 	 */
 	private function remove_custom_logo() {
 		remove_theme_mod( 'custom_logo' );
+		// For FSE themes.
+		delete_option( 'site_logo' );
```

Comment:
This fixes the seemingly unrelated test. Syncing `custom_logo` with `site_logo` is supposed to be done though in the source code, I think something in the tests setup breaks the hooks in `site-logo.php`.

I commented the `block-template.php` changes in this PR and it was still failing, so I think maybe it's related to the actual changes in the fixtures.

Reply:
Indeed 🤔  My other idea was to switch back to the default theme in the teardown of the `block-template.php` tests, rather than staying on the FSE test theme.

Reply:
I've done that now (0780be0ab23451cf163460d9a968028e9bfb0e0d) and reverted yours (8d00e55ab7f3e51e8587dab151477d168ae6b611) -- hope that's okay. I felt it'd be better to contain changes in `block-template.php`. If we want to test the custom/site logo stuff for FSE things, we should add explicit test cases there.